kleva-j/iReporter
# Feature/#162502567/create account using a database

#### What does this PR do?
- Enables user signup using a database.

#### Description of the task to be completed
- Validate user details.
- Hash user password.
- Create a new user using a database.

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162502567/create-account-using-a-database
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162502567](https://www.pivotaltracker.com/story/show/162502567)
